tests/Functional/FileFromMemoryTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Mailgun\Tests\Functional;
+
+/**
+ * Add attachment with file from memory
+ *
+ * @author Wim Verstuyf <wim.verstuyf@codelicious.be>
+ */
+class FileFromMemoryTest extends \PHPUnit_Framework_TestCase
+{
+    public function testAddFileFromMemory()
+    {
+        $fileValidator = function($files) {
+            $fileChecks = [
+                ['name'=>'attachment[0]', 'contents'=>'File content', 'filename' => 'file1.txt'],
+                ['name'=>'attachment[1]', 'filename' => 'mailgun_icon1.png']
```

Comment:
You have to test the contents of attachment[1], right? Create a new fixture text file and make sure the contents match the expected values.